sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/ReadSchemaSuite.scala

```diff
@@ -163,6 +168,7 @@ class VectorizedParquetReadSchemaSuite
 
 class MergedParquetReadSchemaSuite
   extends ReadSchemaSuite
+  with AddColumnIntoTheMiddleTest
```

Comment:
If we want to add `AddColumnIntoTheMiddleTest` to all of above, maybe we can just let `ReadSchemaSuite` extend `AddColumnIntoTheMiddleTest`.

Reply:
Hi, @viirya . `CSV` doesn't support this.